### ISSUE TITLE
The 2.0.6 GUI doesn't show the FC-Flasher Tab when connected to v2 FC

### ISSUE THREAD

**Issue by tobiasbischoff** [fedorcomander/kiss-gui#3]
It should, or? Im on a mac.

**Comment by mihauOne87** [comment]
For me works

**Comment by shermanater** [comment]
I couldn't get the v2 to work on my mac. worked fine in windows but cant get the v1 to flash.

**Comment by brent2658** [comment]
is there a gui for kiss v2 on the mac ?
update:- my firewall was blocking the silicon labs UART driver update for MAC works great now

**Comment by DroneShotFPV** [comment]
Upgraded to 2.0.6 today from 2.0.0. Was running 1.32 firmware and 119d ESC firmware. Upgraded to 2.0.6 as 2.0 wouldn't work with 1.34rc FC firmware. Once I did this, after flashing to 1.34 and using 2.0.6, when I connect a battery to the quad, I can no longe rload ANY tabs in the GUI. ESC's will not flash either, gives "FAILURE: No response from FC!". I also randomly get "your firmware is older than supported by this GUI" message, but it's verified and ACTIVATED on 1.34rc. I am stuck, and cannot update ESC's, or do anything when battery is connected. Verified on both Windows 7 pro PC and Windows 10 Pro PC.
